### PR TITLE
Fix link to EF6 docs

### LINKED ISSUE #19189: Link for "Entity Framework - Code-Based Configuration" is broken
Link for "Entity Framework - Code-Based Configuration": https://docs.microsoft.com/en-us/ef6/fundamentals/configuring/code-based --> is not working

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f376ed07-b726-292f-eb33-7a918337367b
* Version Independent ID: 817b265f-0769-80e1-b42c-e1b0d05ff13b
* Content: [ASP.NET Core and Entity Framework 6](https://docs.microsoft.com/en-us/aspnet/core/data/entity-framework-6?view=aspnetcore-3.1)
* Content Source: [aspnetcore/data/entity-framework-6.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/data/entity-framework-6.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by serpent5**
Fixes #19189.

### CODE REVIEW

File: aspnetcore/data/entity-framework-6.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ author: rick-anderson
 description: Entity Framework 6.3 and later works with ASP.NET Core 3.1 and later.
 ms.author: riande
 ms.custom: mvc
-ms.date: 7/24/2020
+ms.date: 7/14/2020
```
Comment (serpent5) [inline aspnetcore/data/entity-framework-6.md:7]: Somehow, this date was in the future.

File: aspnetcore/data/entity-framework-6.md [hunk 2]
```diff
@@ -4,14 +4,14 @@ author: rick-anderson
...
 no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
 uid: data/entity-framework-6
 ---
 # ASP.NET Core and Entity Framework 6
 
-[Entity Framework](/ef/ef6/) 6.3 and later works with ASP.NET Core 3.1 and later. For more information, see the blog post [Announcing Entity Framework 6.3 with .NET Core Support](https://devblogs.microsoft.com/dotnet/announcing-entity-framework-6-3-preview-with-net-core-support/).
+[Entity Framework](/ef/ef6) 6.3 and later works with ASP.NET Core 3.1 and later. For more information, see the blog post [Announcing Entity Framework 6.3 with .NET Core Support](https://devblogs.microsoft.com/dotnet/announcing-entity-framework-6-3-preview-with-net-core-support).
```
Comment (serpent5) [inline aspnetcore/data/entity-framework-6.md:13]: I've seen `@scottaddie` comment on a couple of PRs re trailing slashes so I cleaned these up too.

Comment (guardrex) [reply aspnetcore/data/entity-framework-6.md:13]: Trailing slashes are ok if the server/site has a 301/302 redirect to it (i.e., avoid a network round-trip). In this case, that's exactly the situation, so I recommend reverting it here.

File: aspnetcore/data/entity-framework-6.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/data/entity-framework-6.md:7]: I'm not surprised ... Rick has a *Tardis*. :smile:

File: aspnetcore/data/entity-framework-6.md (hunk 2 above)
Comment (serpent5) [reply aspnetcore/data/entity-framework-6.md]: @guardrex Thanks for the explanation.

**Pull request merged**